### DISCUSSION TITLE
Chart with multiple Bars per month and a Mysql View

### DISCUSSION THREAD

**Discussion by Sebastiangperez** [filamentphp/filament#10870] [General]
I like to  know how to make a chart widget with filters and multiple bars per month , also it will be nice if i can switch between bars, like hide or show only one, i mean multiple datasets in the same graph.

**Comment by Android63** [comment]
Maybe try the Apex charts plugin: https://filamentphp.com/plugins/leandrocfe-apex-charts

**Comment by Sebastiangperez** [comment]
@Android63 Thanks , i already did this , i followed a tutorial and worked like a charm.

**Reply by Android63** [reply]
Good to know @Sebastiangperez. Could you link the tutorial or show your code for the benefit of others facing the same issue please?

**Reply by alphayowakarindi** [reply]
@Sebastiangperez Could you please link the tutorial you used?
